src/elements/x-accounts/x-accounts.css

```diff
@@ -115,6 +115,32 @@ x-account x-amount {
     }
 }
 
+x-accounts .add-address {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    align-items: center;
+    padding: 8px;
+    font-weight: bold;
+    opacity: 0.7;
+    transition: opacity 250ms ease;
+    border-radius: .5rem;
+}
+
+x-accounts .add-address:hover,
+x-accounts .add-address:focus {
+    opacity: 1;
+}
+
+x-accounts .add-address [identicon-plus] {
+    display: block;
+    width: 60px;
+    height: 60px;
```

Comment:
I think it would look better if it would be the same size as the identicons.

Reply:
I tried that, and it looked massive! The designs for the new Safe use much smaller identicons, so it works there, but in the current account list I think it's too big.

Reply:
With the new Safe layout, the address list will change to the Vue component (which also needs to be adapted with this new icon), and I think then we can make it the same size.

Reply:
Smaller size looks okay imo